.github/workflows/build.yml

```diff
@@ -31,18 +29,115 @@ jobs:
           path: .gradle
           key: ${{ runner.os }}-gradle--${{ hashFiles('**/build.gradle', '**/settings.gradle', '**/gradle.properties') }}
 
-      - name: Format with spotless
+      - name: Build with Gradle
         uses: gradle/gradle-build-action@v2
         with:
-          arguments: spotlessApply --build-cache
+          arguments: build --build-cache
 
-      - uses: stefanzweifel/git-auto-commit-action@v4
+      - name: Upload build
+        uses: actions/upload-artifact@v3
         with:
-          commit_user_name: 'Wynntils'
-          commit_user_email: 'admin@wynntils.com'
-          commit_message: 'chore: spotless formatting'
+          name: build
+          path: |
+            forge/build/libs/wynntils-*.MC1.18.2-forge.jar
+            fabric/build/libs/wynntils-*.MC1.18.2-fabric.jar
+            quilt/build/libs/wynntils-*.MC1.18.2-quilt.jar
 
-      - name: Build with Gradle
-        uses: gradle/gradle-build-action@v2
+  changelog:
+    name: Generate Changelog
+    needs: [build]
+    runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ steps.changelog.outputs.tag }}
+      skipped: ${{ steps.changelog.outputs.skipped }}
+      clean_changelog: ${{ steps.changelog.outputs.clean_changelog }}
+      changelog: ${{ steps.changelog.outputs.changelog }}
```

Comment:
Is this a path to the changelog? But how can that be accessed in later steps, since this is run on a separate VM? 🤔

Reply:
Or is it just the changelog as a string? How is it formatted? Can it really be passed on properly to all other steps?

Reply:
The changelog is passed to the desired workflow. Just like it is for Wynntils but for Artemis we control the intermediary workflow that pushes to Modrinth and Curseforge. However, Scyu shared another workflow that does all in one.

Reply:
Ok, I don't know how this is done in Legacy, so you'll have to explain this PR to me like I'm four years old. ;-)

Is it a string? Formatted? How? Markdown?